app/models/user.rb

```diff
@@ -375,6 +375,15 @@ def identity
     @identity ||= UserIdentity.find(uuid)
   end
 
+  def onboarding
+    if Flipper.enabled?(:veteran_onboarding_beta_flow, self) && user_account_uuid.present?
```

Comment:
Is there a reason to have a flipper here? I would bet you don't even need a Flipper on the backend at all

Reply:
We do need a flipper, because we are planning to roll this "onboarding" feature out to a limited subset of users as we iterate on features.

Reply:
sounds good. Could you move the Flipper inside the `VeteranOnboarding` class? Only recommending to keep the complexity of the `User` model itself down

Reply:
Apologies, @bosawt - I re-requested your review before addressing this request.  Working on it.